wgsl/index.bs

```diff
@@ -9814,219 +9821,219 @@ The following are reserved words:
 
 ## Syntactic Tokens ## {#syntactic-tokens}
 
-A <dfn>syntactic token</dfn> is a sequence of special characters, used:
+A <dfn>syntactic token</dfn> is a sequence of special code points, used:
 * to spell an expression operator, or
 * as punctuation: to group, sequence, or separate other grammar elements.
 
 <div class='syntax' noexport='true'>
   <dfn for=syntax>and</dfn> :
 
-    | `'&'`
+    | `'&'` (Code point: `U+0026`)
```

Comment:
Do these need the `Code point:` prefixes or could the following suffice?

```
| `'&'`   // (`U+0026`)
```

Reply:
@dj2 My initial attempt was this way but I found code point version to be a bit more readable, and also, down the line, we might improve these rule specifications to be a bit more inlining of symbols where they are referenced and this could resolve to the comment symbol or double divide. Though I am neutral to further comments that take extractor aspect into consideration too.